modal_proto/api.proto

```diff
@@ -1269,6 +1270,14 @@ message Resources {
   GPUConfig gpu_config = 4;
 }
 
+// Used in RESOURCE_EXHAUSTED errors to differentiate
+// retryable and non-retryable version of that error status.
+// Lifted from googleapis as an alternative to using the google-common-protos PyPi package.
+// ref: github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto
```

Comment:
We've added `googleapis-common-protos` to `setup.cfg` in this PR. Is that intended? If so, can you update comment?